train.gemspec

```diff
@@ -25,6 +25,7 @@ Gem::Specification.new do |spec|
 
   spec.required_ruby_version = '>= 2.0'
 
+  spec.add_dependency 'addressable', '~> 2.5'
```

Comment:
The pin here on 2.5 is going to conflict with Chef which is pinned to 2.4 due to another gem. Does this need to be 2.5 or would 2.4 work?

Reply:
Hmm, addressable 2.4 is from end of 2015. Maybe I was required to bump the gem while working on a project for Ruby 2.5 support, but maybe I remember wrong. 

For sure a lot of fixes happened since 2.4. I don't know whether the change to addressable gem is even smart when not used with 2.5

Reply:
We're currently waiting on Travis to release this gem so we can pull in the latest addressable to our overall stack: https://github.com/travis-ci/gh/issues/33